include/zephyr/posix/unistd.h

```diff
@@ -55,6 +55,153 @@ pid_t getpid(void);
+								(CONFIG_MAX_PTHREAD_KEY_COUNT), \
+								(-1L))
+#define __z_posix_sysconf_SC_THREAD_STACK_MIN		  PTHREAD_STACK_MIN
+#define __z_posix_sysconf_SC_THREAD_THREADS_MAX		  (-1L)
+#define __z_posix_sysconf_SC_RE_DUP_MAX			  (-1L)
+#define __z_posix_sysconf_SC_RTSIG_MAX			  (-1L)
+#define __z_posix_sysconf_SC_SEM_NSEMS_MAX		  (-1L)
+#define __z_posix_sysconf_SC_SEM_VALUE_MAX		  COND_CODE_1(CONFIG_PTHREAD_IPC,     \
+								      (CONFIG_SEM_VALUE_MAX), \
+								      (-1L))
+#define __z_posix_sysconf_SC_SIGQUEUE_MAX		  (-1L)
+#define __z_posix_sysconf_SC_STREAM_MAX			  (-1L)
+#define __z_posix_sysconf_SC_SYMLOOP_MAX		  (-1L)
+#define __z_posix_sysconf_SC_TIMER_MAX			  CONFIG_MAX_TIMER_COUNT
+#define __z_posix_sysconf_SC_TTY_NAME_MAX		  (-1L)
+#define __z_posix_sysconf_SC_TZNAME_MAX			  (-1L)
+#define __z_posix_sysconf_SC_ADVISORY_INFO		  (-1L)
+#define __z_posix_sysconf_SC_BARRIERS			  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_ASYNCHRONOUS_IO		  (-1L)
+#define __z_posix_sysconf_SC_CLOCK_SELECTION		  (-1L)
+#define __z_posix_sysconf_SC_CPUTIME			  (-1L)
+#define __z_posix_sysconf_SC_FSYNC			  (-1L)
+#define __z_posix_sysconf_SC_IPV6			  (-1L)
+#define __z_posix_sysconf_SC_JOB_CONTRO			  (-1L)
+#define __z_posix_sysconf_SC_MAPPED_FILES		  (-1L)
+#define __z_posix_sysconf_SC_MEMLOCK			  (-1L)
+#define __z_posix_sysconf_SC_MEMLOCK_RANGE		  (-1L)
+#define __z_posix_sysconf_SC_MEMORY_PROTECTION		  (-1L)
+#define __z_posix_sysconf_SC_MESSAGE_PASSING		  (-1L)
+#define __z_posix_sysconf_SC_MONOTONIC_CLOCK		  (-1L)
+#define __z_posix_sysconf_SC_PRIORITIZED_IO		  (-1L)
+#define __z_posix_sysconf_SC_PRIORITY_SCHEDULING	  (-1L)
+#define __z_posix_sysconf_SC_RAW_SOCKETS		  (-1L)
+#define __z_posix_sysconf_SC_READER_WRITER_LOCKS	  (-1L)
+#define __z_posix_sysconf_SC_REALTIME_SIGNALS		  (-1L)
+#define __z_posix_sysconf_SC_REGEXP			  (-1L)
+#define __z_posix_sysconf_SC_SAVED_IDS			  (-1L)
+#define __z_posix_sysconf_SC_SEMAPHORES			  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_SHARED_MEMORY_OBJECTS	  (-1L)
+#define __z_posix_sysconf_SC_SHELL			  (-1L)
+#define __z_posix_sysconf_SC_SPAWN			  (-1L)
+#define __z_posix_sysconf_SC_SPIN_LOCKS			  (-1L)
+#define __z_posix_sysconf_SC_SPORADIC_SERVER		  (-1L)
+#define __z_posix_sysconf_SC_SS_REPL_MAX		  (-1L)
+#define __z_posix_sysconf_SC_SYNCHRONIZED_IO		  (-1L)
+#define __z_posix_sysconf_SC_THREAD_ATTR_STACKADDR	  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_THREAD_ATTR_STACKSIZE	  COND_CODE_1(CONFIG_PTHREAD_IPC,	  \
+								      (SUPPORTED_POSIX1_VERSION), \
+								      (-1L))
+#define __z_posix_sysconf_SC_THREAD_CPUTIME		  (-1L)
+#define __z_posix_sysconf_SC_THREAD_PRIO_INHERIT	  (-1L)
```

Comment:
This would probably be more accurate
```suggestion
#define __z_posix_sysconf_SC_PRIO_INHERIT		  _POSIX_VERSION
```

All Zephyr threads typically inherit the permissions assigned by their parent thread.